content/en/blog/2025/autonomus_testing_with_antithesis.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Autonomous Testing of etcd's Robustness
+author: "Marek Siarkowicz, Google & Craig Alfieri, Antithesis"
+draft: true
+---
+
+As a critical component of many production systems, including Kubernetes, the etcd project's first priority is reliability.
+Ensuring consistency and data safety requires our project contributors to continuously improve testing methodologies.
+In this article, we will describe how we used advanced simulation testing to uncover subtle bugs,
+validate the robustness of our releases, and increase our confidence in etcd's stability.
+We'll share our key findings and how they have improved etcd.
+
+## Enhancing etcd's Robustness Testing
+
+Many critical software systems depend on etcd to be correct and consistent, most notably as the primary datastore for Kubernetes.
+After some issues with the v3.5 release, the etcd maintainers developed a new After some issues with the v3.5 release,
```

Comment:
```suggestion
After some issues with the v3.5 release,
```

Looks like there's some leftover text on this line.

Reply:
Thanks, done